client/app/email-domain/email-domain.service.js

```diff
@@ -0,0 +1,15 @@
+angular.module('services').service(
+  'EmailDomain',
```

Comment:
Otherwise you could just use `OvhApiEmailDomain` in your controller and get rid of this service. What's your feeling about that?

Reply:
Sure. That's possible :)
Just thought that a service would be better if we are going to have more APIs being used. Please let me know you thoughts

Reply:
That's the `ovh-api-services` purpose actually:)

Reply:
Got it (y)
I will get it done :)

Reply:
This is done :)